lib/segment/tests/integration/sparse_vector_index_search_tests.rs

```diff
@@ -570,6 +570,36 @@ fn sparse_vector_persistence_test() {
     let path = segment.current_path.clone();
     drop(segment);
 
+    // persistence using rebuild.
+    // for appendable segment vector index has to be rebuilt
```

Comment:
please a comment for immutable segment RAM

Reply:
Did it, https://github.com/qdrant/qdrant/pull/3154/commits/dbd70b7263b2c5c81498810fcdfef33a7ebc4a95